packages/react/src/patterns/sub-patterns/PictogramItem/PictogramItem.js

```diff
@@ -47,7 +47,7 @@ const PictogramItem = ({
         <ContentItem
           heading={heading}
           copy={copy}
-          cta={{ style: 'text', ...cta }}
+          cta={cta && { style: 'text', ...cta }}
```

Comment:
Maybe I'm missing something, but I think this is redundant.

https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/react/src/patterns/sub-patterns/ContentItem/ContentItem.js#L48-L56

Reply:
As this pattern only supports the `text` style, we need to pass it as a prop inside of the component, and due to this, even if the another `CTA` props are not passed, it will render an incomplete `CTA`.

Reply:
ah, I think I see what you're saying. Interesting dilemma.